Makefile

```diff
@@ -27,15 +27,15 @@ list-modules:
 
 .PHONY: gotest
 gotest:
-	$(MAKE) for-all CMD="make test"
+	@$(MAKE) for-all CMD="make test"
 
 .PHONY: golint
 golint:
-	$(MAKE) for-all CMD="make lint"
+	@$(MAKE) for-all CMD="make lint"
 
 .PHONY: gomod-download-all
 gomod-download-all:
-	$(MAKE) for-all CMD="make mod-download-all"
+	@$(MAKE) for-all CMD="make mod-download-all"
```

Comment:
what is the difference?

Reply:
Doesn't print out the `make` invocation

```
$ make gomod-download-all
/Library/Developer/CommandLineTools/usr/bin/make for-all CMD="make mod-download-all"
running make mod-download-all in all modules...
running make mod-download-all in ./pkg/exporter/sumologicexporter
...
```

vs

```
$ make gomod-download-all
running make mod-download-all in all modules...
running make mod-download-all in ./pkg/exporter/sumologicexporter
```